modules/users/data/data.yaml

```diff
@@ -16,26 +16,7 @@ groups:
                  'ALL = (ALL) NOPASSWD: /usr/sbin/service jobchron *',
                  'ALL = (ALL) NOPASSWD: /usr/bin/puppet *',
                  'ALL = (ALL) NOPASSWD: /root/ssl-certificate',
-                 'ALL = (ALL) NOPASSWD: /bin/journalctl *']
-  mediawiki-roots:
```

Comment:
groups should really be kept even if unused, because the GIDs were in use and reusing GIDs is poor practise.

Reply:
Will handle later